internal/cli/global_opts.go

```diff
@@ -48,6 +48,11 @@ func (opts *GlobalOpts) ConfigOrgID() string {
 	return opts.OrgID
 }
 
+// ConfigMongoShellPath returns the MongoDB Shell path.
+func (opts *GlobalOpts) ConfigMongoShellPath() string {
+	return config.MongoShellPath()
+}
```

Comment:
[nit] let's keep this helpers focused, for now it'll be better to create a new Opts to embed, `MongoshellOpts` works well